src/Cryptol/Eval/Reference.lhs

```diff
@@ -1707,13 +1708,23 @@ and integers.
 These functions capture the interactions with rationals.
 
 
-This just captures a common pattern for binary floating point primitives.
+These just capture common patterns for unary, binary, and trinary floating
```

Comment:
I think the word for an operator with 3 arguments is "ternary"?

Reply:
You're right, I'm not sure why I though "trinary" sounded more natural. (It turns out that [trinary is a word](https://en.wiktionary.org/wiki/trinary), but much less common.) I'll change it to ternary (and rename the `*Tri*` functions to `*Tern*`).